README.md

```diff
@@ -1,3 +1,28 @@
-## About
+# About
 
-This is a (very) lightweight discord clone.
+Liscord is a (very) lightweight discord clone.
+
+# Requirements
+
+* Python 3.x
+
+# Installation
+
+* git clone https://github.com/raqibahmed/liscord
+
+```bash
+pip install -r requirements.txt
```

Comment:
It'll be better to suggest the user to install the dependencies in a separate virtual environment so that their system doesn't get polluted with the project's dependencies.